elasticsearch/src/main/scala/com/snowplowanalytics/stream/loader/clients/ElasticsearchBulkSender.scala

```diff
@@ -101,7 +100,7 @@ class ElasticsearchBulkSender(
    * This BulkHandler is added to change endpoints of bulk api request to add
    * document type to them. This change is made in order to continue to support ES 6.x
    */
-  implicit object CustomBulkHandler extends Handler[BulkRequest, BulkResponse] with Logging {
+  implicit object CustomBulkHandler extends Handler[BulkRequest, BulkResponse] {
```

Comment:
Don't we care about logging?

Reply:
The `Logging` trait got removed from elastic4s [in this commit](https://github.com/sksamuel/elastic4s/commit/3404e5ab13c0d0c75800ca66c84cdaf877867041) and it got replaced with using slf4j directly.  So this line change is necessary and safe.  It does not change what gets logged.